File: mcachedb/python/create/mcachedb_create.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-
#
#	create/mcachedb_create.py
#
#					Jul/29/2014
#
import memcache
#
# ----------------------------------------------------------------
print ("*** 開始 ***")
#
mc = memcache.Client(['127.0.0.1:21201'])
mc.flush_all()
#
mc.set('t1521', '{"name":"新潟","population":37561,"date_mod":"2003-4-24"}')
mc.set('t1522', '{"name":"長岡","population":46382,"date_mod":"2003-6-11"}')
mc.set('t1523', '{"name":"新発田","population":51643,"date_mod":"2003-7-2"}')
mc.set('t1524', '{"name":"上越","population":15624,"date_mod":"2003-8-14"}')
mc.set('t1525', '{"name":"糸魚川","population":37692,"date_mod":"2003-9-17"}')
mc.set('t1526', '{"name":"加茂","population":58619,"date_mod":"2003-10-27"}')
mc.set('t1527', '{"name":"三条","population":27698,"date_mod":"2003-11-24"}')
mc.set('t1528', '{"name":"佐渡","population":58751,"date_mod":"2003-10-7"}')
mc.set('t1529', '{"name":"柏崎","population":65125,"date_mod":"2003-12-14"}')
mc.set('t1530', '{"name":"村上","population":34247,"date_mod":"2003-2-6"}')
mc.set('t1531', '{"name":"十日町","population":17238,"date_mod":"2003-6-8"}')
mc.set('t1532', '{"name":"五泉","population":48921,"date_mod":"2003-5-7"}')
#
print ("*** 終了 ***")
# ----------------------------------------------------------------

File: mcachedb/ruby/create/mcachedb_create.rb
#! /usr/bin/ruby
# -*- coding: utf-8 -*-
#
#	mcachedb_create.rb
#
#						Jul/08/2015
#
require 'dalli'
require 'json'
#
load '/var/www/data_base/common/ruby_common/json_manipulate.rb'
load '/var/www/data_base/common/ruby_common/mcache_manipulate.rb'
# ------------------------------------------------------------
puts "*** 開始 ***"
#
dc = Dalli::Client.new('ubuntu_1504:21201')
#
#
cache["t1521"] = json_gen_proc("新潟",98271,"2006-4-27")
cache["t1522"] = json_gen_proc("長岡",74865,"2006-1-3")
cache["t1523"] = json_gen_proc("新発田",25617,"2006-3-14")
cache["t1524"] = json_gen_proc("上越",84521,"2006-9-3")
cache["t1525"] = json_gen_proc("糸魚川",47189,"2006-8-9")
cache["t1526"] = json_gen_proc("加茂",25164,"2006-7-18")
cache["t1527"] = json_gen_proc("三条",34695,"2006-6-21")
cache["t1528"] = json_gen_proc("佐渡",42735,"2006-3-14")
cache["t1529"] = json_gen_proc("柏崎",54628,"2006-5-22")
cache["t1530"] = json_gen_proc("村上",71385,"2006-9-17")
cache["t1531"] = json_gen_proc("十日町",85312,"2006-8-26")
cache["t1532"] = json_gen_proc("五泉",76348,"2006-7-18")
#
#
puts "*** 終了 ***"
# ------------------------------------------------------------

